pkg/transformer/kubernetes/kubernetes.go

```diff
@@ -640,9 +640,10 @@ func (k *Kubernetes) Undeploy(komposeObject kobject.KomposeObject, opt kobject.C
 			//FIXME: gracePeriod is nil
 			err = rpDeployment.Stop(namespace, t.Name, TIMEOUT*time.Second, nil)
 			if err != nil {
-				return err
```

Comment:
@cdrage , if we returned the error, kompose will not be able to keep doing its job (as discussed in issue, its there only in the case of kompose down)